src/scripts/bot-move.js

```diff
@@ -0,0 +1,30 @@
+import {GAME_SYMBOLS} from "./libraries";
+import {winLines} from "./calculate-winner";
+const bestSquareIndex = 4;
+
+function getWinIndex(squareArray, symbol) {
+    for (let i = 0; i < winLines.length; i++) {
+        if (!winLines[i].some((el) => squareArray[el] === symbol)) {
+            const nextTurns = winLines[i].filter(el => {
+                return squareArray[el] === null;
+            })
+            if (nextTurns.length === 1) {
+                return nextTurns[0];
+            }
+        }
+    }
+    return null;
+}
+
+function getRandomIndex(squareArray) {
+    if (!squareArray[bestSquareIndex]) {
+        return bestSquareIndex;
+    } else {
+        return squareArray.indexOf(null);
+    }
+
+}
+
+export function bestBotMove(squaresArray) {
```

Comment:
rename method, it should be a verb not a noun

Reply:
Renamed to `calculateBotMove`